docs/glossary.md

```diff
@@ -5,4 +5,52 @@ hide_title:  false
 title: Glossary
 ---
 
-Here are list of Nym terminology nad their definitions.
\ No newline at end of file
+**Nym Technologies SA** - Official company name
+
+**Nym** - in-text name of the organisation
+
+**NYM** - The token
+
+**Address** -  
+
+**Ack** - Short for "acknowledgement" 
+
+**Blockchain** - A distributed database that maintains a continuously growing list of ordered records, called blocks
+
+**Coconut** - 
+
+**CLI** - A command-line interface (CLI) is a text-based user interface (UI) used to run programs, manage computer files and interact with the computer.
+
+**Credentials**
+
+**Centralisation** - Refers to the The concentration of control of an activity or organisation under a single authority
+
+**Delegate** - 
+
+**Delegating NYM** - Same as bonding but by non operators and as a proof of their trust on a node operator providing a good quality of service 
```

Comment:
If you're referring to something being the same as bonding then we should include a definition of bonding!